source/dev/idea_list.md

```diff
@@ -0,0 +1,148 @@
+# Google Summer of Code 2021
+Breakthrough Energy Sciences (BES) has applied to serve as a mentoring organization for Google Summer of Code (GSoC).
+The GSoC program offers student a stipend to work on open source code during the summer. 
+If you are interested in the program [here](https://summerofcode.withgoogle.com/) you can find more information.
+
+If you are a student and new to GSoC read the following [guide](https://google.github.io/gsocguides/student/).
+If you are new to BES [here](https://science.breakthroughenergy.org/) more about our organization.
+
+
+## Community and Communication
+We are active on Slack which is the best way to reach us.
+[Join us here](http://science.breakthroughenergy.org/#get-updates), introduce yourself, discuss your potential project in the #ideas channel, and let us know if you have any questions.
+
+If you need to reach us via email, here you go: sciences@breakthroughenergy.org
+
+Please review our [code of conduct](code_of_conduct.md) before joining the community.
+## Application
+If you are a student and applying for GSoC find bellow the idea list and explore the projects we would like to accomplish with you this summer.
+
+When applying please provide:
+
+- A project proposal in markdown following the [Elements of a Quality Proposal guide](https://google.github.io/gsocguides/student/writing-a-proposal#elements-of-a-quality-proposal)
+- Motivation why you choose this particular project?
+- Your resume with relevant experience in PDF format
+## Idea list
+If you can’t find your ideal project on this list,
+but you have something else in mind that would enhance our open source project,
+let's discuss it the #ideas channel on Slack.
+For each project we provided various scales to adapt to your skill, ambition or individual situation.
+
+Also keep in mind when planning out your work that for each project we expect you to follow our [contribution guide](contribution_guide.md).
+### Project 1. - Simulation Data Download Manager
+To run energy system simulations we need input data such as electrical demand, hydro flow, solar irradiance, and wind speed time series data among others. See here for [more details](https://github.com/Breakthrough-Energy/PreREISE#2-gather-data-for-simulation). 
+Currently the data is collected by using the package when needed. There is still a lot of manual labor involved in collecting and cleaning the data. Besides that the data source or API’s may change.
+
+The goal of this project is to encapsulate the existing code for downloading external resources into a single library and CLI.
```

Comment:
First time we use the CLI acronym so we should define it...

... into a single library and Command Line Interface (CLI).

No need to define it in the doc in subsequent uses.